charts/enforcer-k8s/templates/rbac.yaml

```diff
@@ -0,0 +1,22 @@
+# necessary in order to read the secret containing the TLS config and any image pull secrets
```

Comment:
```suggestion
# necessary in order to read the secret containing the TLS config
```

image pull secrets will have to be handled later with a clusterrole / clusterrolebinding, since presumably it will need to read secrets that exist in other namespaces if we want to enforce on namespaces other than the one this is deployed to